tcf_website/tests/test_instructor_edgecases.py

```diff
@@ -0,0 +1,161 @@
+        """Test average difficulty for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
+
+        self.assertTrue(
+            self.instructor.average_rating_for_course(self.course) is
+            None)
+
+    def test_average_enjoyability_for_course_no_reviews(self):
+        """Test average enjoyability for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
+
+        self.assertTrue(
+            self.instructor.average_enjoyability_for_course(self.course) is
+            None)
+
+    def test_average_instructor_rating_for_course_no_reviews(self):
+        """Test average instructor rating for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
+
+        self.assertTrue(
+            self.instructor.average_instructor_rating_for_course(
+                self.course) is None)
+
+    def test_average_recommendability_for_course_no_reviews(self):
+        """Test average recommendability for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
+
+        self.assertTrue(
+            self.instructor.average_recommendability_for_course(
+                self.course) is None)
+
+    def test_average_hours_for_course_no_reviews(self):
+        """Test average hours for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
+
+        self.assertTrue(
+            self.instructor.average_hours_for_course(
+                self.course) is None)
+
+    def test_average_reading_hours_for_course_no_reviews(self):
+        """Test average reading hours for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
+
+        self.assertTrue(
+            self.instructor.average_reading_hours_for_course(
+                self.course) is None)
+
+    def test_average_writing_hours_for_course_no_reviews(self):
+        """Test average reading hours for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
+
+        self.assertTrue(
+            self.instructor.average_writing_hours_for_course(
+                self.course) is None)
+
+    def test_average_group_hours_for_course_no_reviews(self):
+        """Test average group hours for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
+
+        self.assertTrue(
+            self.instructor.average_group_hours_for_course(
+                self.course) is None)
+
+    def test_average_other_hours_for_course_no_reviews(self):
+        """Test average other hours for a particular course when there are no reviews."""
+
+        self.review1.delete()
+        self.review2.delete()
+
+        self.assertTrue(
```

Comment:
I was thinking we should change all of these assertTrue's to assertEqual's since if the test fails assertTrue does not lend any information as to what the two sides of the equality are whereas assertEqual does.